### PR TITLE
Html forms

### PR DESCRIPTION
**Pull request by Aiden-Peace300**
image

### CODE REVIEW

File: html-forms/index.html [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+    <!--Opening tag of the body element-->
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>html-links</title>
+  </head>
+  <!--Closing tag of the body element-->
+  <body>
+    <!--Opening tag of the body element-->
+    <h1>Coding Form!</h1>
```
Comment (shawnkost) [inline html-forms/index.html:13]: Your `form` tag should start right here.

File: html-forms/index.html [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+    <label for="email">Email</label><br />
+    <input type="email" id="email" name="email" />
+    <br />
+
+    <label for="Coding_id">Why is coding cool?<br /></label>
+    <select id="Coding_id" name="coding_options">
```
Comment (shawnkost) [inline html-forms/index.html:19]: `id`s should always be lowercase

File: html-forms/index.html [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+      <option value="select">Select an option</option>
+      <option value="interesting">Because it's interesting</option>
+      <option value="fun">Because it's fun</option>
+      <option value="powerful">Because it's powerful</option>
+    </select>
+
+    <p>Favorite Language (choose one)</p>
+
+    <form>
```
Comment (shawnkost) [inline html-forms/index.html:28]: Needs to be moved up so that all of your form control elements are inside

**Review by shawnkost — CHANGES_REQUESTED** [review]
Some changes requested here Aiden!

File: html-forms/index.html (hunk 1 above)
Comment (Aiden-Peace300) [reply html-forms/index.html:13]: below line 13 correct?

**Comment by Aiden-Peace300** [comment]
image

**Pull request merged**